docs/get-details/ethereum_to_algorand.md

```diff
@@ -0,0 +1,231 @@
+title: From Ethereum to Algorand
+
+This guide is meant for Ethereum **developers** who want to switch to Algorand to benefit from its numerous advantages: faster block time (less than 4s), immediate finality (a transaction can never be reverted once in a block), very low fees, and high throughput, while still being highly secure.
+
+It is focused on **dApp developers** (as opposed to end users and node runners).
+
+If you want to find the equivalent of any Ethereum specific notion/tool/service, just search on this page and you will most likely find the equivalent notion/tool/service on Algorand.
+
+# Main High-Level Differences
+
+In this section, we highlight the main differences between Ethereum and Algorand.
+
+## Accounts and Smart Contracts
+
+Both Ethereum and Algorand are account-based blockchains supporting smart contracts.
+
+On Ethereum, smart contracts are actually represented by accounts, while on Algorand smart contracts and accounts are two different objects.
+Concretely:
+
+* Ethereum's **Externally-owned accounts (EOA)** correspond to Algorand accounts. They are both represented by an **address**.
+    * Example of Ethereum address: 
+        * User-friendly representation: `0x65e9980679DE55744f386aa1999307f1687A92f9`
+        * Raw address: 20 bytes
+    * Example of Algorand address: 
+        * User-friendly representation: `QD3BO4RMWXBOZIPHTGGB3RSKSOAKOHM2HGN7QDZXH4ECBGJRIU3AHHC3JU` 
+        * Raw address: 32 bytes
+* Ethereum's **contract accounts** correspond to Algorand **application ID**, which are 64-bit integers.
+    * Example of Ethereum contract account: 
+        * User-friendly representation: `0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d`
+        * Raw: 20 bytes
+    * Example of Algorand application ID: 
```

Comment:
you should point out the note that an algorand address can be calculated. Just add something like for an actual address see note below

Reply:
done above